shared/checks/oval/mcafee_antivirus_definitions_updated.xml

```diff
@@ -0,0 +1,49 @@
+<def-group oval_version="5.10">
+  <definition class="compliance" id="mcafee_antivirus_definitions_updated" version="1">
+    <metadata>
+      <title>Confirm Existence and Permissions of System Log Files</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>Verify that McAfee AntiVirus definitions have been updated.</description>
+    </metadata>
+
+    <criteria>
+      <criterion comment="Check if McAfee AntiVirus definitions have been updated" test_ref="test_mcafee_antivirus_definitions_updated" />
+    </criteria>
+
+  </definition>
+
+  <unix:file_object id="mcafee_dat_files_mtime" comment="McAfee definition files" version="1">
+    <unix:path datatype="string">/opt/NAI/LinuxShield/engine/dat</unix:path>
+    <unix:filename datatype="string" operation="pattern match">^.*\.dat$</unix:filename>
+  </unix:file_object>
+
+  <local_variable id="variable_mcafee_dat_files_mtime" version="1" comment="Age of file for each McAfee definition file." datatype="int">
+      <time_difference format_2="seconds_since_epoch">
+        <object_component object_ref="mcafee_dat_files_mtime" item_field="m_time"/> 
+      </time_difference>
+  </local_variable>
+
+  <ind:variable_test check="all" check_existence="all_exist" id="test_mcafee_antivirus_definitions_updated" version="1" comment="McAfee AntiVirus definitions have been updated">
+    <ind:object object_ref="object_mcafee_antivirus_definitions_updated" />
+    <ind:state state_ref="state_mcafee_antivirus_definitions_updated" />
```

Comment:
Are the objects and states always named after the name of tests?

Nitpick:
Would be more clear if this object was called something like "antivirus definitions modified time".
And state called "antivirus definition timeout", or "max modified time".

Reply:
> Are the objects and states always named after the name of tests?

They are done more often than not, but updated them per your suggestion.